spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -23,13 +28,13 @@
   end
 
   describe "create" do
+    let!(:group) { create(:group) }
 
     it "creates a skill (with full parameters)" do
-      group = create(:group)
       post :create, format: :json, skill: {
-                                            name: "created",
-                                            featured: true,
-                                            group: group.name
+                                             name: "created",
```

Comment:
I'm wondering if a newline character is the culprit on this error. It's not a huge deal, but maybe you can try putting the hash options on a single line? (e.g. { name: "name", group: "group", featured: true }